lib/commons/color/get-background-color.js

```diff
@@ -65,9 +77,48 @@ function calculateObscuringAlpha(elmIndex, elmStack) {
 }
 
 function elmOutsideParent(elm, bgElm, bgColor) {
```

Comment:
It doesn't. This loops over the entire stack of elements beneath it. The function name isn't all that clear though. These things don't even have to be ancestor elements. They can be any two elements.